examples/v1beta1/pipelineruns/4808-regression.yaml

```diff
@@ -92,4 +92,4 @@ spec:
     name: result-test
   params:
   - name: RESULT_STRING_LENGTH
-    value: "3000"
+    value: "2000"
```

Comment:
is there a reason why you change the length here? just curious

Reply:
The signature and the size of the certificate takes up approximately 1k. More details here https://github.com/tektoncd/community/blob/main/teps/0089-nonfalsifiable-provenance-support.md#signing-results.
Hence we reduce the number of bytes for the actual result.

Reply:
With sidecar logs, we would not have to reduce this right? This is because of termination message extraction method?

Reply:
I think we should have a test for sidecar logs which tests the maximum size of the result with sidecars.
With that we will still have to reduce the size of the result (i.e. maximum result) to accommodate the signatures of the results.

Reply:
https://github.com/tektoncd/pipeline/issues/6363
Done.